tests/engines/registries/acceptance/overview/topbar-test.ts

```diff
@@ -0,0 +1,206 @@
+        const reg = server.create('registration', {
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+        });
+
+        await visit(`/${reg.id}/`);
+
+        assert.dom('[data-test-social-sharing-button]').isVisible();
+
+        await triggerEvent('[data-test-social-sharing-button]', 'mouseenter');
+        assert.dom('.ember-bootstrap-tooltip .tooltip-inner').hasText(
+            t('registries.overview.tooltips.share').toString(),
+        );
+
+        assert.dom('[data-test-sharing-options]').isNotVisible();
+        await click('[data-test-social-sharing-button]');
+        assert.dom('[data-test-sharing-options]').isVisible();
+
+        ['Email', 'Linkedin', 'Twitter', 'Facebook'].forEach(medium =>
+            assert.dom(`[data-test-share-registration="${medium}"]`).isVisible());
+    });
+
+    test('forks dropdown works', async assert => {
+        const reg = server.create('registration', {
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+        });
+
+        await visit(`/${reg.id}/`);
+
+        assert.dom('[data-test-forks-dropdown-button]').isVisible();
+
+        await triggerEvent('[data-test-forks-dropdown-button]', 'mouseenter');
+        assert.dom('.ember-bootstrap-tooltip .tooltip-inner').hasText(
+            t('registries.overview.tooltips.fork').toString(),
+        );
+
+        assert.dom('[data-test-forks-dropdown-options]').isNotVisible();
+        await click('[data-test-forks-dropdown-button]');
+        assert.dom('[data-test-forks-dropdown-options]').isVisible();
+
+        assert.dom('[data-test-go-to-forks-view]').isVisible();
+        assert.notOk(Boolean(reg.forkIds.length));
+
+        await click('[data-test-fork-registration]');
+
+        reg.reload();
+        assert.ok(Boolean(reg.forkIds.length));
+    });
+
+    test('state description has correct text', async assert => {
+        for (const [state, stateInfo] of Object.entries(registrationStates)) {
+            const reg = server.create('registration', {
+                registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+                currentUserPermissions: Object.values(Permission),
+            }, stateInfo.trait);
+
+            await visit(`/${reg.id}/`);
+
+            assert.dom('[data-test-state-button]').hasText(t(`registries.overview.${state}.text`).toString());
+
+            if (!stateInfo.initiallyOpened) {
+                await click('[data-test-state-button]');
+            }
+
+            if (stateInfo.hasAdminActions) {
+                assert.dom('[data-test-state-admin-actions]').isVisible();
+            }
+            assert.dom('[data-test-state-description-short]').hasText(
+                t(`registries.overview.${state}.short_description`).toString(),
+            );
+            if (!reg.embargoEndDate) {
+                // ember-i18n test helper t does not support passing in options object.
```

Comment:
[This isn't true](https://github.com/jamesarosen/ember-i18n/blob/v5.1.0/addon-test-support/index.js#L4), but [the types we wrote don't include the second parameter](https://github.com/fabmiz/ember-osf-web/blob/ffce3d542a39503c0e6e1afff587ed6358bf01b2/types/ember-i18n/test-support/index.d.ts#L4). If you update the type to
```ts
export function t(translation: string, interpolations?: Record<string, unknown>): SafeString;
```
you should be able to pass interpolations like normal.